docs/README-developers.md

```diff
@@ -96,3 +96,92 @@ More details about tests can be found in the [tests folder](../tests/).
 
 *Note*: These run integration tests that invoke `c2rust transpile`.
 `cargo test` only runs unit tests and doc tests as of now.
+
+## Documentation
+
+Local documentation can be built with the normal `cargo doc` and `cargo doc --open`.
+
+### `rustc`
+
+However, as the `#![feature(rustc_private)]` crates
+are not normal dependencies managed through `cargo`,
+accessing the documentation for those is trickier.
+
+#### `rust-src`
+
+`rust-src` is the `rustup` component containing `rustc`'s source code.
+This is needed to view `rustc`'s source when developing.
+
+`rust-src` is included in our [`rust-toolchain.toml`](../rust-toolchain.toml),
+so it will automatically be installed.  Otherwise,
+
+```sh
+rustup component add rust-src
+```
+
+will work.
+
+#### `rustc-docs`
+
+`rustc-docs` (not `rust-docs`) is the `rustup` component
+containing the `rustdoc` docs for `rustc`.
+This is different from `rust-docs`, the `rustup` component
+containing the `rustdoc` docs for `std`, various books on Rust,
+and other miscellaneous documentation.
+
+Unfortunately, the `rustc-docs` and `rust-docs` components
+are currently mutually incompatible, as they install to the same location:
+`share/doc/rust/html`.
+See [`rust-lang/rustup#3196`](https://github.com/rust-lang/rustup/issues/3196).
+
+Thus, `rustc-docs` is not included in [`rust-toolchain.toml`](../rust-toolchain.toml)
+to avoid any build errors with `rust-docs`, which is installed by default.
+
+To install `rustc-docs`, run
+
+```sh
+rustup component remove rust-docs
+rustup component add rustc-docs
```

Comment:
Does this need a particular toolchain version, i.e. `rustup +nightly-... component ...`, or does it pick it up from the toolchain file? If it picks it up from the toolchain file, you should mention that it has to be run from under this project to pick up the right version.

Reply:
It detects the right one from `rust-toolchain.toml`.  I'll add that it has to be run from inside this repo.

Reply:
Fixed in 676b81362f76000b82deb7630811b03ffe65c12b.